.github/workflows/unittests.yaml

```diff
@@ -0,0 +1,85 @@
+name: unit tests
+
+on:
+  push: # run on every push or PR to any branch
+  # pull_request:
+
+env:
+  SOLR_CORE: derrida
+
+jobs:
+  python-unit:
+    name: python unit tests
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: mariadb
+        env:
+          MYSQL_DATABASE: derrida
+          MYSQL_ROOT_PASSWORD: derrida
+        ports:
+          - 3306:3306
+      solr:
+        image: solr:6.6
+        ports:
+          - 8983:8983
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          # Run on python 3.6 since 3.5 is no longer supported
+          python-version: 3.6
+
+      # Perform installations
+      - name: Install dependencies
+        run: |
+          pip install -r ci/ci-requirements.lock
+          pip install codecov
+          sudo apt-get install ruby-sass
+
+      # Setup local settings
+      - name: Configure test settings as local settings
+        run: |
+          cp ci/testsettings.py derrida/local_settings.py
+          python -c "import uuid; print('SECRET_KEY = \'%s\'' % uuid.uuid4())" >> derrida/local_settings.py
+
+      # configure solr core per haystack directions
+      # create a new core with basic configs
+      # copy core config to local machine and remove from docker vm
+      # use haystack to customize the config, copy it back, and reload the core
+      - name: Setup solr core
+        run: |
+          docker exec ${{ job.services.solr.id }} /bin/bash -c "/opt/solr/bin/solr create -c $SOLR_CORE -n basic_config"
+          docker cp ${{ job.services.solr.id }}:/opt/solr/server/solr/derrida/conf /tmp/derrida-conf
+          python manage.py build_solr_schema --configure-directory=/tmp/derrida-conf
+          docker exec ${{ job.services.solr.id }} /bin/bash -c "rm -rf /opt/solr/server/solr/derrida/conf"
+          docker cp /tmp/derrida-conf ${{ job.services.solr.id }}:/opt/solr/server/solr/derrida/conf
+          python manage.py build_solr_schema --configure-directory=/tmp/derrida-conf  --reload-core $SOLR_CORE
```

Comment:
I guess the thing I couldn't figure out was to remove the existing config, and then you can reload the core from haystack, not the docker container?

Reply:
yeah, it took a little bit of local testing for me to figure out a working solution — the basic config uses `managed-schema` but haystack still uses old-style `schema.xml`; if you pull it down haystack renames the managed one, but copying it back up without removing it leaves in place

We could have done the curl command to reload the container, but I did the lazy thing and just used haystack to regenerate the schema again and then run the reload command.